docs/shared/migration/overview.md

```diff
@@ -0,0 +1,189 @@
+# Migrating existing code bases
+
+## What you’ll accomplish
+
+Migrating into an Nx workspace can seem intimidating. While every codebase is unique, we can offer recommendations for how to proceed based on the Nrwl team’s years of experience. The key to success is an incremental approach. You don’t need to migrate your entire codebase at once. Find a good target for migration, move it into your Nx workspace, get it working and stable, and go from there.
+
+This document will introduce you to the Nx workspace, help you get one created, and point you to some particular configuration that you may need for your codebase.
+
+## Create a workspace
+
+To get started, you’ll need an Nx workspace. While Nx offers several preset configurations for your workspace, we recommend starting migration with an `empty` workspace. This will allow you to have finer control over the configuration and creation of the applications and libraries in your workspace.
+
+To create an empty workspace:
+
+```bash
+npx create-nx-workspace@latest
+```
+
+Select `empty` when prompted:
+
+```bash
+? What to create in the new workspace (Use arrow keys)
+❯ empty             [an empty workspace]
+```
+
+## Exploring your workspace
+
+Take a tour of your [Nx workspace](/{{framework}}/workspace/overview). There are some important areas to know about as you migrate.
+
+### apps
+
+The `apps` directory is the place where your top-level applications will be stored. You should always begin your migration journey by adding an application.
+
+For Angular applications:
+
+```bash
+nx generate @nrwl/angular:application my-application
+```
+
+For React applications:
+
+```bash
+nx generate @nrwl/react:application my-application
+```
+
+There are a lot of options when creating your application. If you want to follow Nx recommendations, you can accept the defaults. If you have a well-established codebase, you can configure those options at the time of application generation. You can find documentation for these options for the different frameworks here:
+
+- [Angular](/angular/plugins/angular/schematics/application)
+- [React](/react/plugins/react/schematics/application)
+
+You may also find it useful to use the [Nx Console](/nx-console) in Visual Studio Code. This will give you a visual way to generate your application with all of the options laid out in front of you.
```

Comment:
```suggestion
You may also find it useful to use the [Nx Console](/{{framework}}/cli/console) in Visual Studio Code. This will give you a visual way to generate your application with all of the options laid out in front of you.
```